packages/react-graphql-universal-provider/src/request-id-link.ts

```diff
@@ -0,0 +1,10 @@
+import {setContext} from 'apollo-link-context';
+
```

Comment:
I don't understand why some source file names are hyphenized and some are camelized. How do you decide which one is correct?

Reply:
React components => camel case
Rest of the javascript things => hypenized